clusterloader2/testing/load/config.yaml

```diff
@@ -19,10 +19,7 @@
 {{$MEDIUM_STATEFUL_SETS_PER_NAMESPACE := DefaultParam .MEDIUM_STATEFUL_SETS_PER_NAMESPACE 1}}
 {{$ENABLE_CHAOSMONKEY := DefaultParam .ENABLE_CHAOSMONKEY false}}
 {{$CHECK_IF_PODS_ARE_UPDATED := DefaultParam .CL2_CHECK_IF_PODS_ARE_UPDATED true}}
-{{$ENABLE_PVS := DefaultParam .CL2_ENABLE_PVS true}}
 {{$DISABLE_DAEMONSETS := DefaultParam .CL2_DISABLE_DAEMONSETS false}}
```

Comment:
Before, this flag allowed us to disable  daemonset. In new module it's not used currently. Could you fix that please :)?